lib/rx/client.rb

```diff
@@ -142,7 +142,7 @@ def get_tracking_history_rx(id)
     #
     def post_refill_rxs(ids)
       if (result = perform(:post, get_path('rxrefill'), ids, get_headers(token_headers)))
-        increment_refill(ids.size)
+        increment_refill
```

Comment:
This should remain as `increment_refill(ids.size)` so that we get an accurate count of the number of Rx refills that have been processed.